index.html

```diff
@@ -63,6 +63,99 @@
       </div>
 
       <!--End Navbar-->
+
+      <!-- popup -->
```

Comment:
- [ ] kindly take out all comments from code, it is generally a considered good [HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) practice